content/_help/trouble-signing-in/face-or-touch-unlock._en.md

```diff
@@ -58,6 +67,20 @@ Go to [secure.login.gov](https://secure.login.gov/) and click on "Create an acco
 2. You will receive an email. Click on the link in the email to confirm your email address.
 3. Create a password. It must be at least 12 characters long.
 4. On the “Secure your account” page, select face or touch unlock.
-5. Enter a device nickname for face or touch unlock.
-6. Upon clicking the “continue” button, you will then be prompted to authenticate using face or touch unlock. 
-7. Follow the steps to set up another authentication method. 
+5. On the face or touch unlock set up page, click on the “continue” button. You will then be prompted to authenticate using face or touch unlock.
+    **Important**: Depending on your device, you may be prompted for additional information such as a PIN, pattern, or QR code. Follow the directions on your screen to finish setting up face or touch unlock. The following are screenshots of possible prompts you may encounter:
+
+    <div class="grid-row grid-gap">
+        <div class="tablet:grid-col">
+            <img alt="Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot of a screen lock PIN prompt" src="{{ site.baseurl }}/assets/img/help/face-touch-unlock/android-screen-lock.png" />
```

Comment:
These images end up being quite small when arranged horizontally on the website, vs. the original Google Doc. Would be curious for feedback from UX folks. At the very least, I'd think we should make the images clickable so someone could easily view them in their original size.